_sponsors/2022-07-08-tworocksoftware.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: base
+
+hidden: false
+level: "Bronze"
+name: "Two Rock Software"
+logo: "/static/img/sponsors/Two-Rock-Software-Logo.svg"
```

Comment:
Is this file already in the repo?

I'm on my phone so it's not obvious to check

Reply:
Confirmed, I see it. Merging.

Reply:
Ya, for some reason it was already in the repo but the sponsors file for Two Rock didn't make it from the 2021 repo.